.github/services/s3/0_minio_s3/action.yml

```diff
@@ -31,7 +31,10 @@ runs:
         AWS_ACCESS_KEY_ID: "minioadmin"
         AWS_SECRET_ACCESS_KEY: "minioadmin"
         AWS_EC2_METADATA_DISABLED: "true"
-      run: aws --endpoint-url http://127.0.0.1:9000/ s3 mb s3://test
+      run: |
+        aws --endpoint-url http://127.0.0.1:9000/ s3 mb s3://test
+        aws --endpoint-url http://127.0.0.1:9000/ s3api put-bucket-versioning --bucket test --versioning-configuration Status=Enabled
```

Comment:
Hi, how about adding new tests instead of changing the existing ones? This way, we can ensure that our code works on services with versioning enabled or not.

Reply:
Okay, I'm a bit hesitant about this as well.

Reply:
Thank you!